src/components/TasksTable/index.jsx

```diff
@@ -100,6 +100,9 @@ const assignments = Object.values(ASSIGNEE);
     flex: 1,
     whiteSpace: 'nowrap',
   },
+  adventurousButton: {
+    paddingRight: 0,
```

Comment:
There seems to be an issue when hovering on the button (no white space on the right):

![screen shot 2018-08-27 at 5 33 36 pm](https://user-images.githubusercontent.com/3766511/44687564-8fea3880-aa1f-11e8-919b-9d011f93a3a9.png)

Reply:
Oh, I missed that!
I’ll have a look at this.

Reply:
No worries :)

Reply:
@helfi92 
I think I have fixed this. 
Please review it again :)

Before hovering:

![screen shot 2018-08-29 at 11 44 29 am](https://user-images.githubusercontent.com/24795489/44769057-f5a20600-ab80-11e8-91d1-e6b05d6565b1.png)

On hovering: 

![screen shot 2018-08-29 at 11 43 53 am](https://user-images.githubusercontent.com/24795489/44769091-12d6d480-ab81-11e8-8fbc-1f827901f772.png)